### PR TITLE
fix: register 2nd page

### PR DESCRIPTION
**Pull request by tokitouq**

### CODE REVIEW

File: backend/django_core/apps/pages/forms/user.py [hunk 1]
```diff
@@ -84,9 +87,50 @@ class FirstRegisterForm(forms.Form):
...
+                "placeholder": "enter the code",
+                "class": "h-12 w-full rounded-xl border-[0.4vw] border-primary-500 bg-transparent px-5 text-base font-medium outline-none !ring-0 transition-all placeholder:text-white/50 focus:border-primary-400 md:h-[3.125vw] md:rounded-[0.75vw] md:border-[0.2vw] md:px-[1vw] md:text-[1.1vw]",
+            }
+        ),
+        help_text="if you didn’t receive the code, check your spam folder. Or use the resend button",
+    )
+
+    def clean(self):
+        cleaned_data = self.cleaned_data
+        username = self.cleaned_data["username"]
```
Comment (baseplate-admin) [inline backend/django_core/apps/pages/forms/user.py:124]: 🤣

File: backend/django_core/apps/pages/forms/user.py [hunk 2]
```diff
@@ -84,9 +87,50 @@ class FirstRegisterForm(forms.Form):
...
+        username_pattern = r"^[a-zA-Z0-9_-]+#[0-9]{4}$"
```
Comment (baseplate-admin) [inline backend/django_core/apps/pages/forms/user.py:125]: 🤣☠️

File: backend/django_core/apps/pages/forms/user.py [hunk 3]
```diff
@@ -84,9 +87,50 @@ class FirstRegisterForm(forms.Form):
...
+        pattern = re.compile("^[a-zA-Z0-9_-]+#[0-9]{4}$")
```
Comment (baseplate-admin) [inline backend/django_core/apps/pages/forms/user.py:126]: ☠️☠️🔪

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: backend/django_core/apps/pages/forms/user.py (hunk 3 above)
Comment (tokitouq) [reply backend/django_core/apps/pages/forms/user.py]: Well I can explain :')

**Pull request merged**